images/linux/scripts/SoftwareReport/SoftwareReport.CachedTools.psm1

```diff
@@ -28,6 +28,33 @@ function Get-ToolcacheGoVersions {
     return Get-ChildItem $toolcachePath -Name | Sort-Object { [Version]$_ }
 }
 
+function Build-GoEnvironmentTable {
```

Comment:
We shouldn't hardcode exact Go versions here. For tool-cache, it is enough to just iterate through all versions in `$toolcachePath = Join-Path $env:AGENT_TOOLSDIRECTORY "Go"`.

See existing function in Windows: https://github.com/actions/virtual-environments/blob/main/images/win/scripts/SoftwareReport/SoftwareReport.CachedTools.psm1#L36

Also, we need to add `arch` property for Go like in Windows